recipes/libgpiod/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+import platform
+
+required_conan_version = ">=1.33.0"
+
+class LibgpiodConan(ConanFile):
+    name = "libgpiod"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://git.kernel.org/pub/scm/libs/libgpiod/libgpiod.git/"
+    license = "LGPL-2.1-or-later"
+    description = "C library and tools for interacting with the linux GPIO character device"
+    topics = ("conan", "gpio", "libgpiod", "libgpiodcxx", "linux")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "enable_bindings_cxx": [True, False],
+               "enable_tools": [True, False]}
+    default_options = {"shared": False,
+                       "enable_bindings_cxx": False,
+                       "enable_tools": False}
+    
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def validate(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("only Linux is supported")
+        if tools.Version(platform.release()) < "4.8":
```

Comment:
Follow liburing as example: https://github.com/conan-io/conan-center-index/blob/master/recipes/liburing/all/conanfile.py#L54

Reply:
still no luck. I guess I'll revisit this tomorrow